From 85419ca09d225630cd1740e554bfb65c560e3904 Mon Sep 17 00:00:00 2001
From: Gary Lin <glin@suse.com>
Date: Wed, 21 Dec 2022 14:49:34 +0800
Date: Tue, 08 Oct 2024 04:18:22 +0800
Subject: [PATCH] Exclude '-march=native' from all archs

Since we do not want our build results to depend on the build machine's CPU,
remove it from HOST_CPPFLAGS.
fixes https://bugzilla.opensuse.org/show_bug.cgi?id=1231368

Signed-off-by: Gary Lin <glin@suse.com>
Signed-off-by: Bernhard M. Wiedemann <bwiedemann suse.com>
---
 src/include/defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/include/defaults.mk b/src/include/defaults.mk
index 9da5182..94502ed 100644
--- a/src/include/defaults.mk
+++ b/src/include/defaults.mk
@@ -70,7 +70,7 @@ override SOFLAGS = $(_SOFLAGS) \
 		   $(call family,SOFLAGS)
 
 HOST_ARCH=$(shell uname -m)
-ifneq ($(HOST_ARCH),ia64)
+ifeq (0,1)
 	HOST_MARCH=-march=native
 else
 	HOST_MARCH=
-- 
2.35.3

